elements/lisk-transactions/src/base_transaction.ts

```diff
@@ -244,7 +250,7 @@ export abstract class BaseTransaction {
 		} = validateSignature(
```

Comment:
there's another call to this function in line 395 I think that might need to be updated?